src/xcode/ENA/ENA/Resources/Localization/bg.lproj/Localizable.strings

```diff
@@ -523,11 +523,11 @@
 
 "ExposureSubmission_PhoneDetailsDomestic" = "За обаждания на територията на Германия.\nОбаждането е безплатно.";
 
-"ExposureSubmission_PhoneNumberForeign" = "+49 30 498 75401";
+"ExposureSubmission_PhoneNumberForeign" = "+49 30 498 75402";
```

Comment:
@marcussc Seems like translation delivery takes numbers in to account too so it mapped on the basis of "de" locale